engines/ags/detection_tables.h

```diff
@@ -4091,6 +4188,9 @@ const AGSGameDescription GAME_DESCRIPTIONS[] = {
 	GAME_ENTRY_EN("anightmareonduckburg", "ANoD.exe", "c9cd3e2cdd295c33e62cfa97b14461e9", 91549100),
 	GAME_ENTRY_EN("anightthatwouldntend", "Game.exe", "0241777c2537fc5d077c05cde10bfa9f", 14559632),
 	GAME_ENTRY_EN("anighttoremember", "the trials.exe", "a01a9639ce30bdcd5bf82e528b51fa06", 9643993),
+	GAME_ENTRY_EN("animalagency", "AnimalAgency.exe", "7db052bc30700d1f30f5330f5814f519", 613308708),  // Win
+	GAME_ENTRY_EN("animalagency", "AnimalAgency.ags", "39bab349175b4b7fec0a6b13b701d6eb", 610273552),  // Linux
+	GAME_ENTRY_EN("animalagency", "game.ags", "39bab349175b4b7fec0a6b13b701d6eb", 610273552),  // Mac
```

Comment:
That is a very generic name. Can this entry be enhanced?

Reply:
You mean the full name or the identifier? Well the itch.io page is simply "Animal Agency" but the title screen says "Uma Breakdown Animal Agency", we can use something like that

Reply:
He means the name of the files used for detection I think.
In general using files with a generic name causes issues with false-positive detection when a game from another engine could be detected as an unknown variant of an AGS game by mistake for example.
However for AGS games there is sometimes no choice as there is no other files to use for detection. And furthermore using generic names is not really an issue thanks to 6d2a62fe72a773fca04a05baf2c174df1200f686. So I think you can ignore this request as well as the similar one below.

Reply:
Yes, that's what I referring to. "game.ags" is quite a generic name, which could lead to false positives. Thanks for the update, @criezy

Reply:
Ok, as @criezy stated this is not really an issue and there are already hundreds of games that are detected using generic filenames in the AGS detection table... can I resolve this conversation then?